### PR TITLE
Change geth command to continuously stream blocks

### PR DESCRIPTION
**Pull request by agatsoh**
## Description

Fixes: 
This is part of the fixes that would go in for https://github.com/raiden-network/light-client/issues/2487

Please, describe what this PR does in detail:
- If it's a bug fix, detail the root cause of the bug and how this PR fixes it.
- If it's a new feature, describe the feature this PR is introducing and design decisions.
- If it's a refactoring, describe why it is necessary. What are its pros and cons in respect to the previous code, and other possible design choices.

### CODE REVIEW

**Comment by agatsoh** [comment]
@palango FYI

File: docs/custom-setups/private_net_tutorial.rst [hunk 1]
```diff
@@ -65,21 +65,21 @@ Prepare a file ``genesis.json`` with the following content.
...
-"ByzantiumBlock": 0,
-"eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
-"ConstantinopleBlock": 0,
-"PetersburgBlock": 0
-},
-"alloc": {},
-"difficulty" : "0x1",
-"gasLimit"   : "0x9880000"
-}
+ "config": {
```
Comment (palango) [inline docs/custom-setups/private_net_tutorial.rst]: This formatting is weird, why add spaces there?

Comment (agatsoh) [reply docs/custom-setups/private_net_tutorial.rst]: If we dont add spaces it will not be properly formatted

Comment (palango) [reply docs/custom-setups/private_net_tutorial.rst]: But then do it consistenly, after `"config": {` there is a new level of indentation. and usually indention is at least 2 spaces.

Comment (agatsoh) [reply docs/custom-setups/private_net_tutorial.rst]: Ok I just saw it. I have given consistent spacing after `config`. I also had a look at this file as a whole. On this file go to the top right corner 3 dots. Click on `view file`. It is showing up correctly. Rest I am also trying to run this locally and see if documentation comes out correctly but atleast in github it is turning out correctly.

Comment (palango) [reply docs/custom-setups/private_net_tutorial.rst]: Screen Shot 2021-08-10 at 14 22 55
This is what I see in the diff, and this is **not** correctly indented.

**Comment by palango** [comment]
@agatsoh Can you please fix this so we can merge it?

**Comment by agatsoh** [comment]
@palango please have a look again. I rebased it on top of develop now because there were some merge conflicts.

**Review by palango — APPROVED** [review]

**Pull request merged**